package.json

```diff
@@ -77,6 +79,7 @@
     "pg": "^8.9.0",
     "pg-query-stream": "^4.3.0",
     "pino": "^8.10.0",
+    "rate-limiter-flexible": "^2.4.1",
```

Comment:
This is not needed anymore right?

Reply:
Still needed for specifying which store client to use, in our case, Redis. A direct dependency of the primary package of rate limiter too.